packages/itwinui-css/src/divider/divider.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-divider {
+  background-color: var(--iui-color-border-subtle);
+  flex-shrink: 0;
```

Comment:
`flex: none` might be better, since we don't want it grow or shrink

Reply:
Ooh, good idea, changed!